src/server.js

```diff
@@ -60,15 +60,17 @@ module.exports = () => {
     app.use(passport.session())                                     // Init sessions
 
     app.use(cookieParser(process.env.EXPRESS_SESSION_SECRET))       // Parses cookies using the env SS
-    app.use(helmet())                                               // Express security
+    app.use(helmet({
+        contentSecurityPolicy: false
+    }))                                                             // Express security
```

Comment:
Why was CSP disabled? The iframe is loaded locally and doesn't interact with the Express server. This should be kept on to mitigate XSS attacks.

Reply:
CSP disallows loading of scripts